common/http/src/main/java/io/helidon/common/http/Http.java

```diff
@@ -1890,6 +1883,15 @@ public static ZonedDateTime parse(String text) {
             }
         }
 
+        /**
+         * Last recorded timestamp.
+         *
+         * @return timestamp
+         */
+        public static ZonedDateTime timestamp() {
+            return time;
+        }
```

Comment:
Clever!

Reply:
Wanted to keep the performance impact to a minimum, and we do not care about seconds that much in this case.